frontend/src/ScoreBox.js

```diff
@@ -243,15 +243,15 @@ class ScoreBox extends Component {
             }
             return a ? 1 : -1
           },
-          Cell: props => {
+          Cell: ({ column, original: key, value }) => {
```

Comment:
are you trying to destructure key out of original? This isn't the right syntax for that. This renames original as key, you want { original: { key }} instead...